### PR TITLE
September Release RKE2 dev-v2.6-0922 patches

### PR DESCRIPTION
**Pull request by nikolaishields**

### CODE REVIEW

File: channels-rke2.yaml [hunk 1]
```diff
@@ -598,11 +598,11 @@ releases:
   - version: v1.22.13+rke2r1
     minChannelServerVersion: v2.6.3-alpha1
     maxChannelServerVersion: v2.6.99
-    serverArgs: &serverArgs-v1
+    serverArgs: &serverArgs-v122-v1
```
Comment (briandowns) [inline channels-rke2.yaml]: Hyphens?

File: channels-rke2.yaml [hunk 2]
```diff
@@ -733,17 +747,39 @@ releases:
...
         repo: rancher-rke2-charts
         version: v3.23.103
       rke2-calico-crd:
         repo: rancher-rke2-charts
         version: v3.23.103
     featureVersions: *featureVersions-v1
+  - version: v1.23.12+rke2r1
+    minChannelServerVersion: v2.6.4-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: *serverArgs-v123-v1
```
Comment (briandowns) [inline channels-rke2.yaml]: Missing hyphens?

File: channels-rke2.yaml [hunk 3]
```diff
@@ -758,7 +794,42 @@ releases:
...
+      rke2-calico-crd:
+        repo: rancher-rke2-charts
+        version: v3.24.102
+    featureVersions: *featureVersions-v1
+  - version: v1.24.6+rke2r1
+    minChannelServerVersion: v2.6.7-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: *serverArgs-v124-v1
+    agentArgs: *agentArgs-v124-v1
+    charts: &charts-v124-v2
```
Comment (briandowns) [inline channels-rke2.yaml]: Missing hyphens

**Review by rancher-max — APPROVED** [review]
IT WORKS! LGTM

**Review by matttrach — APPROVED** [review]
LGTM

**Review by brandond — APPROVED** [review]
LGTM. Please squash when or before merging, we don't need all the `WIP` and `Revert` commits in the log.

**Pull request merged**